db-init/Dockerfile

```
@@ -1,8 +1,7 @@
-FROM flyway/flyway:9.20-alpine
+FROM flyway/flyway:9.21-alpine
 
 # hadolint ignore=DL3018
-RUN apk update && apk upgrade && apk --no-cache add libcrypto3 libssl3 openssl  && rm -rf /var/cache/apk/*
-
+RUN apk update && apk upgrade && apk --no-cache add libcrypto3 libssl3 openssl  && rm -rf /var/cache/apk/* && pip3 install --no-cache-dir sqlfluff==2.1.2
```

Separate apk and pip installs:
```suggestion
RUN apk update && apk upgrade && apk --no-cache add libcrypto3 libssl3 openssl  && rm -rf /var/cache/apk/* 
RUN pip3 install --no-cache-dir sqlfluff==2.1.2
```

---

Done